### DISCUSSION TITLE
prose classes other than DEFAULT: bullets in unordered lists are not displayed

### DISCUSSION THREAD

**Discussion by mxs17** [tailwindlabs/tailwindcss#5822] [Help] [answered]
Hello! When I use the DEFAULT `prose` class, I can see the default gray bullet in the `ul > li::before` pseudo-element. I can also change its color via tailwind.config.js as described in the documentation. At the same time, with `prose-sm`, `prose-lg`, etc. classes, that `::before` pseudo-element is not added. Is this by design? If so, what is the correct workaround to style the bullets in unordered lists?

<img width="791" alt="Screenshot 2021-10-18 at 22 11 03" src="https://user-images.githubusercontent.com/32967922/137792357-190cfa84-bdc5-4b3b-a52f-81e199624f40.png">

**Comment by aggreggator** [comment, marked as answer]
Both classes are needed: `prose` (formatting) and `prose-sm` (modifier)

Example:
https://play.tailwindcss.com/13TZabM1TD

You can find it in the docs:
https://github.com/tailwindlabs/tailwindcss-typography#size-modifiers

**Reply by mxs17** [reply]
Thanks!
